### PR TITLE
chore: deployed prod abis

### PR DESCRIPTION
**Pull request by akwritescode**
# Task:

Submit all the production abis and deploy scripts used to deploy to prod

## Description

All prod abis and deploy scripts with the right  params

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## Checklist

- [X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

**Review by aleone — APPROVED** [review]
lgtm

File: packages/hardhat/contracts/core/Controller.sol [hunk 1]
```diff
@@ -476,7 +476,7 @@ contract Controller is Ownable, ReentrancyGuard, IERC721Receiver {
             return 0;
         }
 
-        // add back the bounty amount, liquidators only get reward from liquidation
+        // add back the bounty amount, liquidators onlly get reward from liquidation
```
Comment (alexisgauba) [inline packages/hardhat/contracts/core/Controller.sol:479]:
```suggestion
        // add back the bounty amount, liquidators only get reward from liquidation
```

**Review by alexisgauba — APPROVED** [review]

File: packages/hardhat/contracts/core/Controller.sol (hunk 1 above)
Comment (aleone) [reply packages/hardhat/contracts/core/Controller.sol:479]: I think this maybe should be a separate PR and not merged in yet to ensure we have a commit that matches the deployed bytecode.  We can fix in master afterwards.

Comment (akwritescode) [reply packages/hardhat/contracts/core/Controller.sol:479]: i believe this is on my branch when i actually deployed.

**Pull request merged**